docs/devel/cherry-picks.md

```diff
@@ -64,6 +64,13 @@ hack/cherry_pick_pull.sh upstream/release-3.14 98765
 This will walk you through the steps to propose an automated cherry pick of pull
  #98765 for remote branch `upstream/release-3.14`.
 
+#### Labeling
+
+Ensure your PR contains the label `cherrypick-candidate`.
```

Comment:
Updated with these comments and with comments from [here](https://github.com/kubernetes/kubernetes/pull/23617#issuecomment-203213024).